packages/react-native-sdk/src/components/call/CallControls/ChatButton.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { CallControlsButton } from './CallControlsButton';
+import { theme } from '../../../theme';
+import { Chat } from '../../../icons';
+import { ComponentTestIds } from '../../../constants/TestIds';
+import { Z_INDEX } from '../../../constants';
+
+/**
+ * The props for the Chat Button in the Call Controls.
+ */
+export type ChatButtonProps = {
+  /**
+   * Handler to be called when the chat button is pressed.
+   * @returns void
+   */
+  onPressHandler?: () => void;
+  /**
+   * The unread message indicator to be displayed above on the Chat button.
+   */
+  unreadBadgeCountIndicator?: number;
```

Comment:
```suggestion
  unreadBadgeCount?: number;
```

more easier to understand in my opinion, just an optional suggestion

Reply:
Done ✅